src/GraphQL.Tests/Types/ComplexGraphTypeTests.cs

```diff
@@ -19,7 +19,7 @@ public class ComplexGraphTypeTests
         internal class ComplexType<T> : ComplexGraphType<T> {
             public ComplexType()
             {
-                Name = typeof(T).GetFriendlyName();
+                Name = typeof(T).GetFriendlyName().Replace("<", "Of").Replace(">", "");
```

Comment:
`GetFriendlyName` returns forbidden characters for graph type name so just replace it.